src/main/webapp/components/BuildSnapshotContainer.js

```diff
@@ -35,9 +35,17 @@ export const BuildSnapshotContainer = React.memo((props) =>
         .catch(setData([]));
     }, []);
 
+    let content;
+    if (data.length > 0) {
+      content = data.map(snapshot => <BuildSnapshot buildData={snapshot}/>);
+    } else {
+      content = <span className='loader'>
+          No new revisions to display as of {new Date().toString()}.</span>;
```

Comment:
Is this displaying the date in the user's time zone?

Reply:
Yes!

Reply:
Replaced in this commit 00dabaa